rslib/src/notetype/stock.rs

```diff
@@ -37,6 +37,8 @@ pub fn all_stock_notetypes(tr: &I18n) -> Vec<Notetype> {
         basic_optional_reverse(tr),
         basic_typing(tr),
         cloze(tr),
+        image_occlusion_hide_all_guess_one(tr),
+        image_occlusion_hide_one_guess_one(tr),
```

Comment:
I have added two notetypes because we need to store the values about hideAll boolean. I am thinking about adding one notetype and moving the hideAll boolean to fields of the notetype. That will be easier to switch between two type for current notes. What are the thoughts?